Octokit.Tests.Integration/fixtures/OrganizationsHooksCollection.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Xunit;
+
+namespace Octokit.Tests.Integration.fixtures
```

Comment:
> I already changed the name on the other 2 classes as well should I undo the changes?
Whatever you prefer.  I was trying to avoid you needing to tackle more problems on this PR that were already existing (I also wasnt sure what the repercussions of renaming that folder and or namespace might be) which is why I thought a separate PR gets it off your plate.  But if you've done it then no big deal!